Tests/js/views/assets/ez-view-tests.js

```diff
@@ -36,7 +36,56 @@ YUI.add('ez-view-tests', function (Y) {
                 view.get('subPlainView').get('active'),
                 "The active attribute of the subPlainView should be undefined"
             );
-        }
+        },
+    });
+
+    addEventsTest = new Y.Test.Case({
+        name: "eZ View addEvent tests",
+
+        setUp: function () {
+            this.TestView = Y.Base.create('TestView', Y.eZ.View, [], {
+                _something: function() {},
+            }, {});
+            this.event1 = {'.something': {
+                'tap': '_something'
+            }};
+            this.event2 = {'.whatever': {
+                'tap': '_something'
+            }};
+            this.view = new this.TestView({
+                events: this.event1,
+            });
+        },
+
+        _testAddEventsInDom: function () {
+            this.view._addDOMEventHandlers(this.event2);
+
+            Assert.areSame(
+                this.event1[".something"],
+                this.view.events['.something'],
+                "Event 1 should be present in the view events"
+            );
+            Assert.areSame(
+                this.event2[".whatever"],
+                this.view.events['.whatever'],
+                "Event 2 should be present in the view events"
+            );
+        },
+
+        "Should add events to the DOM event handler": function () {
+            this._testAddEventsInDom();
+        },
+
+        "Should add events to the DOM event handler and attach events": function () {
+            this.view.set('container', '.container');
+            this._testAddEventsInDom();
+
+            Assert.areSame(
+                2,
+                this.view._attachedViewEvents.length,
```

Comment:
this is a protected protected, you can not use it directly. Instead your TestView should subscribe to a real event (like click on something) and you should check that the subscription works as expected.